packages/jest-message-util/src/index.js

```diff
@@ -194,15 +196,45 @@ export const formatStackTrace = (
   testPath: ?Path,
 ) => {
   let lines = stack.split(/\n/);
+  let renderedCallsite = '';
   const relativeTestPath = testPath
     ? slash(path.relative(config.rootDir, testPath))
     : null;
   lines = removeInternalStackEntries(lines, options);
-  return lines
+
+  if (testPath) {
+    const topFrame = lines
+      .join('\n')
+      .trim()
+      .split('\n')[0];
+
+    const parsedFrame = StackUtils.parseLine(topFrame);
+
+    if (parsedFrame) {
+      renderedCallsite = codeFrameColumns(
+        fs.readFileSync(testPath, 'utf8'),
```

Comment:
main thing is that I really don't wanna do this - we have the test in a cache somewhere in memory

Reply:
You are right but unless we share the global state somehow we can't avoid it. We have the file in `jest-runner` and we could possibly put it somewhere as state. Let me think about this for a bit.